packages/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "target": "es5",
```

Comment:
I didn't check but it might be unnecessary to change the `target` from `es6` to `es5`

Reply:
Very true!

Reply:
We can even go up to ES2017 I reckon https://kangax.github.io/compat-table/es2016plus/

Reply:
Then, it should be changed in the original `tsconfig` ?